katsdpdisp/html/client.js

```diff
@@ -286,7 +286,10 @@ function setsignals(){
         handle_data_user_event('inputs');
     }else if (signaltext.slice(0,4)=='holo')
     {
-        handle_data_user_event('holo'+','+signaltext.slice(4));
+        handle_data_user_event('holohh,'+signaltext.slice(4));
+    }else if (signaltext.slice(0,6)=='holohh' || signaltext.slice(0,6)=='holovv' || signaltext.slice(0,6)=='holovh' || signaltext.slice(0,6)=='holovh')
+    {
+        handle_data_user_event(signaltext.slice(0,6)+','+signaltext.slice(6));
```

Comment:
This code will never be reached because it will match the `signaltext.slice(0,4)=='holo'` test above.